addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2130,6 +2140,7 @@ export class OdooEditor extends EventTarget {
                     this.toolbar.style.visibility = show ? 'visible' : 'hidden';
                 }
                 if (show === false) {
+                    this.closeDropDowns();
```

Comment:
Again style coherence, we try to avoid creating functions if they're only going to be used once. Please move the contents of `closeDropdowns` here, prefixed by a comment to say it will close any open dropdown in the toolbar.